panther_hardware_interfaces/include/panther_hardware_interfaces/panther_system_ros_interface.hpp

```diff
@@ -24,17 +24,22 @@
 
 #include <realtime_tools/realtime_publisher.h>
 
+#include <std_msgs/msg/bool.hpp>
+#include <std_srvs/srv/set_bool.hpp>
 #include <std_srvs/srv/trigger.hpp>
 
 #include <panther_msgs/msg/driver_state.hpp>
+#include <panther_msgs/msg/io_state.hpp>
 
+#include <panther_hardware_interfaces/gpio_controller.hpp>
 #include <panther_hardware_interfaces/roboteq_data_converters.hpp>
 
 namespace panther_hardware_interfaces
 {
 
-using TriggerSrv = std_srvs::srv::Trigger;
 using DriverStateMsg = panther_msgs::msg::DriverState;
+using IOStateMsg = panther_msgs::msg::IOState;
+using TriggerSrv = std_srvs::srv::Trigger;
```

Comment:
```suggestion
using DriverStateMsg = panther_msgs::msg::DriverState;
using IOStateMsg = panther_msgs::msg::IOState;
using BoolMsg = std_msgs::msg::Bool;
using SetBoolSrv = std_srvs::srv::SetBoolSrv;
using TriggerSrv = std_srvs::srv::Trigger;
```
and this can be applied later